### DISCUSSION TITLE
windows linux mint dual boot

### DISCUSSION THREAD

**Discussion by bill0549** [linuxmint/discussions#678] [Questions]
I tried to install Linux Mint in dual boot mode, but the installation completely blew out windows.  What did I do wrong?

**Comment by claudiux** [comment]
Are your Windows partitions still there? To find out, install `gparted`, run it and take a screenshot (post it here; the 'partition' to 'flags' columns must be visible).
Close gparted without any other action.

**Reply by bill0549** [reply]
Here's the screen shot

![Screenshot from 2024-11-15 20-03-08](https://github.com/user-attachments/assets/7646d688-5c24-484b-8170-58233f40c942)

**Comment by claudiux** [comment]
No other disk? In this case, you haven't installed Linux Mint in dual boot mode, but have chosen to use the whole disk. All your old operating system is lost.
I sympathize, but the documentation and installation procedure are clear about dual-boot installation.
I hope you've made a backup of your data.

**Comment by bill0549** [comment]
It's not the end of the world. I have recovery disks for the laptop. I'll
start all over from scratch. I've only been using Linux for a couple of
months. I'll copy all my docs and downloads to a USB drive and reinstall
Mint after reinstalling windows. Maybe I'll get it right this time.

Thanks for the help and the speedy response!
